src/reports/filters.php

```diff
@@ -33,7 +33,10 @@ public function get()
 		{
 			ApiError::raiseError(400, JText::_('PLG_API_REPORTS_REPORT_NAME_MISSSING'), 'APIValidationException');
 		}
-		
+
+		$lang = JFactory::getLanguage();
+		$lang->load('', JPATH_ADMINISTRATOR, 'en-GB', true);
```

Comment:
I think this needs to be `$lang->load('plg_api_reports', JPATH_ADMINISTRATOR, 'en-GB', true);` @monikaatvowel

Reply:
refer https://github.com/techjoomla/plg_api_easysocial/blob/master/src/easysocial.php#L34

Reply:
This loads Joomla default language file only

Reply:
@monikaatvowel This will be required for all API plugins. Must be loaded with the com_api component.

cc
@manojLondhe